owncloudApp/src/main/java/com/owncloud/android/files/FileMenuFilter.java

```diff
@@ -289,6 +314,20 @@ private boolean anyFileSynchronizing() {
         return synchronizing;
     }
 
+    private boolean anyFileSynchronizingLookingIntoFilesSync() {
+        boolean synchronizing = false;
+        if (!mFiles.isEmpty()) {
+            for (int i = 0; !synchronizing && i < mFilesSync.size(); i++) {
+                for (int j = 0; !synchronizing && j < mFiles.size(); j++) {
+                    if (mFilesSync.get(i).getFileId() == mFiles.get(j).getId()) {
+                        synchronizing = mFilesSync.get(i).isSynchronizing();
+                    }
+                }
+            }
+        }
+        return synchronizing;
+    }
```

Comment:
Well, in fact, in `mFilesSync` we don't have the file info, only the id. But what we need here is the id, so it's enough to loop through that list. Let me explain it more in the next review comment 😀

Reply:
As far as I know, mFilesSync does not have the file info because we don't send it. AFAIK, we perform a map to send only the Sync info instead of the file and sync info.

Sending all together could get rid of a for loop and the check inside it. So, im not sure 🤔